### PR TITLE
Ensure example .rrd files have description first

### PR DESCRIPTION
**Pull request by emilk**
### What
* Closes  https://github.com/rerun-io/rerun/issues/7175

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7179?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7179?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/7179)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

**Comment by emilk** [comment]
@rerun-bot full-check

File: crates/store/re_entity_db/src/entity_db.rs [hunk 1]
```diff
@@ -466,35 +466,34 @@ impl EntityDb {
...
             .map(|msg| Ok(LogMsg::SetStoreInfo(msg.clone())));
 
-        let time_filter = time_selection.map(|(timeline, range)| {
-            (
-                timeline,
-                ResolvedTimeRange::new(range.min.floor(), range.max.ceil()),
-            )
-        });
+        let data_messages = {
+            let mut chunks: Vec<&Arc<Chunk>> = self.store().iter_chunks().collect();
```
Comment (teh-cmc) [inline crates/store/re_entity_db/src/entity_db.rs:470]: should be possible to apply the time filter before collect()

File: crates/store/re_entity_db/src/entity_db.rs [hunk 2]
```diff
@@ -466,35 +466,34 @@ impl EntityDb {
...
-            .store()
-            .iter_chunks()
-            .filter(move |chunk| {
-                let Some((timeline, time_range)) = time_filter else {
-                    return true;
-                };
+            // Try to roughly preserve the order of the chunks
+            // from how they were originally logged.
+            // See https://github.com/rerun-io/rerun/issues/7175 for why.
+            chunks.sort_by_key(|chunk| chunk.row_ids().next());
```
Comment (teh-cmc) [inline crates/store/re_entity_db/src/entity_db.rs:489]: In practice as of today this is fine, but `row_ids().next()`'s ordering is undefined really.
```suggestion
            chunks.sort_by_key(|chunk| chunk.row_id_range().map(|(min, _)| min);

**Review by teh-cmc — APPROVED** [review]

File: crates/store/re_entity_db/src/entity_db.rs (hunk 2 above)
Comment (emilk) [reply crates/store/re_entity_db/src/entity_db.rs:489]: undefined how?

Comment (emilk) [reply crates/store/re_entity_db/src/entity_db.rs:499]: Ah, they can be unsorted :+1:

Comment (teh-cmc) [reply crates/store/re_entity_db/src/entity_db.rs:499]: The chunk might not be sorted, `row_id_range` makes sure to handle that case.

**Pull request merged**